src/nextjournal/clerk/builder.clj

```diff
@@ -183,8 +187,30 @@
       (expand-paths {:paths-fn `my-paths}))
 #_(expand-paths {:paths ["notebooks/viewers**"]})
 
+(defn compile-css!
+  "Compiles a minimal tailwind css stylesheet with only the used styles included, replaces the generated stylesheet link in html pages."
+  [{:keys [out-path]} docs]
+  (assert (and (= 0 (:exit (sh "which" "npx"))) (= 0 (:exit (sh "npx" "tailwindcss"))))
+          "Clerk's CSS optimizaiton failed: node and tailwind need to be installed. Please run `npm install -D tailwindcss @tailwindcss/typography` and retry.")
+  (spit "tailwind.config.cjs" (slurp (io/resource "stylesheets/tailwind.config.js")))
```

Comment:
Why .cjs and not .js?

Reply:
because of type: module in the package.json. All .js files are treated like ES modules unless they have a .cjs extension.

Reply:
Oh that was with yarn, npx seems happy either way